plasmapy/plasma/grids.py

```diff
@@ -348,7 +342,7 @@ def unit(self):
 
         Raises
         ------
-        ValueError
+        `ValueError`
```

Comment:
I'm totally okay with this.  Just pointing out that it makes no difference when rendering the RTD docs.  For some reason, the `numpydoc` "Raises" section never applies the code syntax when this is done.